packages/node-cli/src/commands/object/import/ImportObjectsAction.js

```diff
@@ -59,20 +66,29 @@ module.exports = class ImportObjectsAction extends BaseAction {
 
 		try {
 			const flags = [];
+			let scriptIdArray;
```

Comment:
If we don't initialize scripIdArray, it can lead to an error later at line 99 when trying to check `scriptIdArray.length`.
We should consider it. Because is making the following call fail
`suitecloud object:import --type usereventscript --scriptid something --destinationfolder /Objects`

Reply:
Fixed this. No need to initialize this. I added line 89 for non-interactive mode that will initialize the scriptIdArray.